src/Wizard/WizardTemplates.js

```diff
@@ -0,0 +1,140 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import cx from 'classnames'
+
+/**
+ * WizardHeader component for Patternfly React
+ */
+export const WizardHeader = ({ children, className, title, ...rest }) => {
+  const classes = cx('wizard-pf-header', className)
+  return (
+    <div className={classes} {...rest}>
+      <h4 className="wizard-pf-title">{title}</h4>
```

Comment:
What are your thoughts on making minimal class changes to better align the UI with what's shown here: http://www.patternfly.org/pattern-library/communication/wizard/

If it's simple enough to have two variations of the header, one for embedded and one for modal, then here are some changes you could make to get the embedded version closer to the pf design example:

- remove the classes `wizard-pf-header` and `wizard-pf-title`. 
- use an `<h2>` instead of <h4>

With these changes, there will still be a border which isn't included on the design example of the embedded wizard, but I'm not seeing where that's getting applied.

Reply:
Cool - I'll be happy to review this on Monday w/ you @jgiardino